demisto_sdk/commands/validate/validate_manager.py

```diff
@@ -849,11 +857,13 @@ def validate_integration(self, structure_validator, pack_error_ignore_list, is_m
                                                        check_is_unskipped=self.check_is_unskipped,
                                                        conf_json_data=self.conf_json_data)
 
-    def validate_script(self, structure_validator, pack_error_ignore_list, is_modified, file_type):
+    def validate_script(self, structure_validator, pack_error_ignore_list, is_modified, file_type, is_added):
```

Comment:
For consistency between integrations, scripts, and playbooks - set the argument order (for all) as:
`self, structure_validator, pack_error_ignore_list, file_type, is_added, is_modified`